src/Listener/EntityChangedListener.php

```diff
@@ -106,4 +112,38 @@ public function preFlush(PreFlushEventArgs $event)
             }
         }
     }
+
+    /**
+     * Pre Persist event callback
+     *
+     * Checks if the entity contains an @Tracked (or derived)
+     * annotation. If so, it will dispatch 'Events::ENTITY_CHANGED'
+     * with the new entity states.
+     *
+     * @param LifecycleEventArgs $event
+     */
+    public function prePersist(LifecycleEventArgs $event)
+    {
+        $em     = $event->getEntityManager();
+        $entity = $event->getEntity();
+
+        if (false === $this->meta_annotation_provider->isTracked($em, $entity)) {
+            return;
+        }
+
+        $mutated_fields = $this->meta_mutation_provider->getMutatedFields($em, $entity, null);
+
+        $this->logger->info(
+            'Going to notify a change (prePersist) to {entity_class}, which has {mutated_fields}',
+            [
+                'entity_class' => get_class($entity),
+                'mutated_fields' => $mutated_fields
+            ]
+        );
+
+        $em->getEventManager()->dispatchEvent(
+            Events::ENTITY_CHANGED,
+            new EntityChangedEvent($em, $entity, null, $mutated_fields)
+        );
```

Comment:
The old one still exists. This name was already used for new entities aswell as updates to already persisted entities, otherwise blamable would not have worked.